Api/CashModuleApi/CashModuleApi/Database.sql

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Is this committed accidentally ? If not, why this sql file has unrelevant content ?